ERCS/erc-1111.md

```diff
@@ -0,0 +1,296 @@
+---
+eip: 1111
```

Comment:
```suggestion
eip: 7629
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.